index.html

```diff
@@ -1,14 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Javascript Homework</title>
-</head>
-<body>
-    <script src="ageCalculator.js"></script>
-    <script src="dogAgeCalculator.js"></script>
-    <script src="housePriceCalculator.js"></script>
-    <script src="startupNameGenerator.js"></script>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Recipe App</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Recipe App</h1>
+    <div id="recipe-container"></div>
+
+    <h2>Add a New Recipe</h2>
+    <form id="recipe-form">
+      <input
+        type="text"
+        id="title"
+        placeholder="Recipe Title"
+        required
+      /><br /><br />
```

Comment:
It's a good practice to add `<label>` tags and to associate them with the input fields. It improves accessibility. For example, now if you start typing in a field, you will lose the text from the placeholder attribute. Some people might forget what the field was about, while a label will always stay above. Another reason to add labels, is that people using screen-readers (software that pronounces what's on a page) would be able to identify the fields better, than if they are just with placeholders.
```suggestion
      <label for="title">Recipe title</label>
      <input
        type="text"
        id="title"
        placeholder="Recipe Title"
        required
      /><br /><br />
```